gwpy/signal/filter_design.py

```diff
@@ -182,7 +183,7 @@ def truncate_transfer(transfer, ncorner=None):
     return out
 
 
-def truncate_impulse(impulse, ntaps, window='hanning'):
+def truncate_impulse(impulse, ntaps, window='hamming'):
```

Comment:
what's the motivation for this change?

Reply:
I'm actually not sure why that's there, I can change it back if you like? (A Hanning window was the default for the old `TimeSeries.whiten` method.)

Reply:
@alurban, yes please, lets keep this PR as clean as possible

Reply:
@duncanmmacleod, ok, I'll make that change shortly

Reply:
This is done with the latest commit (which I've squashed with the main commit)